src/MIDebugEngine/AD7.Impl/AD7BoundBreakpoint.cs

```diff
@@ -65,7 +74,7 @@ public int Delete()
         // Called by the debugger UI when the user is enabling or disabling a breakpoint.
         int IDebugBoundBreakpoint2.Enable(int fEnable)
         {
-            _enabled = fEnable == 0 ? false : true;
+            Enabled = fEnable == 0 ? false : true;
```

Comment:
Technically this isn't quite correct, though it might not matter for the MIEngine -- this method is supposed to set the backend state, which means it should really be updating the backend if necessary, instead of just updating the property.